app/assets/javascripts/Components/dashboard.jsx

```diff
@@ -39,13 +40,27 @@ class Dashboard extends React.Component {
       if (this.state.display_course_summary) {
         // TODO
       } else if (this.state.assessment_type === 'GradeEntryForm') {
-        // TODO
+        $.get({url: Routes.grade_distribution_data_grade_entry_form_path(this.state.assessment_id)}).then(res => {
+
+          let new_labels = ['0 - 5']
+          for (let i = 1; i < 20; i++) {
+            let grade_range = (i * 5 + 1).toString() + " - " + (i * 5 + 5).toString()
+            new_labels.push(grade_range)
+          }
+
+          let new_datasets = [{data: res['grade_distribution']}]
+          let new_data = {labels: new_labels, datasets: new_datasets}
+
+          this.setState({data: new_data})
```

Comment:
Generally end statements with semicolons. I'm commenting here because you'll definitely keep this line of code!